packages/zapp/console/src/components/Literals/LiteralMapViewer.tsx

```diff
@@ -20,13 +22,21 @@ export const LiteralMapViewer: React.FC<{
   className?: string;
   map: LiteralMap | null;
   showBrackets?: boolean;
-}> = ({ className, map, showBrackets = false }) => {
+  showJson?: boolean;
+}> = ({ className, map, showBrackets = false, showJson }) => {
   if (!map) {
     return <NoDataIsAvailable />;
   }
 
   const commonStyles = useCommonStyles();
   const { literals } = map;
+
+  if (!Object.keys(literals).length) {
+    return <NoneTypeValue />;
+  }
+
+  let transformedLiterals = transformLiteralMap(literals);
+
   const mapContent = Object.keys(literals).length ? (
     <ul className={classnames(className, commonStyles.textMonospace, commonStyles.listUnstyled)}>
```

Comment:
Correct, amnything mapContent related will be gone. It's still in there because i wanted to leave the ability to test side by side to visually check for regressions.

Reply:
I'm usually testing against DEMO :)

Reply:
Otherwise we can use before/after screenshots